### DISCUSSION TITLE
Concealer Not Hiding Markup

### DISCUSSION THREAD

**Discussion by Diego-Llanes** [nvim-neorg/neorg#1069] [Q&A]
Hey there I am super new to Neorg and I don't really know what I am doing and could use some help, I am trying to get the concealer to hide away some of the basic markup that I have in my file. Right now the concealer hides away things like todo blocks and headings but not things like bold or italicize. I believe that I am using the correct version of my nerdfont, I have run `brew upgrade font-jetbrains-mono-nerd-font` to make sure that it is the latest. I have also tried `set cole 3` to see if it would further conceal away the markup with no luck.

Any help here is super appriciated! <3

(Also for some reason /Italics/ doesn't work either lol)

<img width="425" alt="Screenshot 2023-09-08 at 5 41 27 PM" src="https://github.com/nvim-neorg/neorg/assets/71251565/678a80d7-5f53-4557-b9fd-abb63158ed27">

**Comment by Diego-Llanes** [comment]
Fixed it sorry! For some reason conceal level wasn't working correctly but it just started working after restarting my terminal. I am using iTerm2 if it is helpful.

**Reply by vhyrro** [reply]
Glad to hear it works now :)
